packages/sfui/frameworks/vue/composables/useScrollable/useScrollable.ts

```diff
@@ -36,30 +36,30 @@ export function useScrollable<TElement extends HTMLElement>(options?: ComputedRe
     { immediate: true, deep: true },
   );
 
-  const getPrevButtonProps = computed(() => {
-    const onClick = () => {
-      scrollable.value?.prev();
-    };
-    return {
-      onClick,
-      disabled: !state.value.hasPrev,
-    };
-  });
+  const goPrev = () => {
+    scrollable.value?.prev();
+  };
+
+  const goNext = () => {
+    scrollable.value?.next();
+  };
+
+  const getPrevButtonProps = computed(() => ({
+    onClick: goPrev,
+    disabled: !state.value.hasPrev,
+  }));
 
-  const getNextButtonProps = computed(() => {
-    const onClick = () => {
-      scrollable.value?.next();
-    };
-    return {
-      onClick,
-      disabled: !state.value.hasNext,
-    };
-  });
+  const getNextButtonProps = computed(() => ({
+    onClick: goNext,
+    disabled: !state.value.hasNext,
+  }));
 
   return {
-    getContainerRef: containerElement,
+    containerRef,
     getPrevButtonProps,
     getNextButtonProps,
+    goNext,
+    goPrev,
```

Comment:
It supposed to be `showNext` and `showPrev`.
```suggestion
    showNext,
    showPrev,
```